go.mod

```diff
@@ -28,9 +28,9 @@ require (
 	github.com/tendermint/tmlibs v0.9.0
 	github.com/tmthrgd/go-hex v0.0.0-20190904060850-447a3041c3bc
 	golang.org/x/crypto v0.0.0-20201208171446-5f87f3452ae9
+	google.golang.org/appengine v1.6.6 // indirect
 	google.golang.org/genproto v0.0.0-20201207150747-9ee31aac76e7
 	google.golang.org/grpc v1.33.2
-	google.golang.org/protobuf v1.25.0
```

Comment:
how did this happen? 🤔

Reply:
I'm not sure at the moment.. I'm not sure why some proto generated files in x/cvm have changed

Reply:
might be something with the environment. Let me double check